packages/dev/core/src/Misc/tools.ts

```diff
@@ -167,29 +165,6 @@ export class Tools {
         return InstantiationTools.Instantiate(className);
     }
 
-    /**
-     * Provides a slice function that will work even on IE
-     * @param data defines the array to slice
-     * @param start defines the start of the data (optional)
-     * @param end defines the end of the data (optional)
-     * @returns the new sliced array
-     */
-    public static Slice<T>(data: T, start?: number, end?: number): T {

```

Comment:
If we remove these from Tools, this is also a breaking change.  Are we okay with this?

Reply:
Yes this is ok as there is a native solution for it

Reply:
What if there is a playground that uses these functions that we are removing? Wouldn't the playground be broken?

Reply:
I only found one instance on the Playground that uses Slice or SliceToArray: https://doc.babylonjs.com/playground?q=Tools.Slice&type=code

Reply:
StringTools is more common though :-/

https://doc.babylonjs.com/playground?q=StringTools&type=code

Reply:
Mark it as deprecated, I will move it to the LTS package as part of the esm work